docs/srt-live-transmit.md

```diff
@@ -208,36 +215,72 @@ explicitly. Note also special cases of the **host** and **port** parts
+  - The *host* part, if specified, can be also used to set the adapter - although in this case **mode=listener** must be set explicitly.
+  - The **port** parameter is not used.
 - **RENDEZVOUS**: the *host* and *port* parts are mandatory.
-    -   The *host* part specifies the remote host to contact.
-    -   The *port* part specifies **both local and remote port**. Note that the local port is this way both listening port and outgoing port.
-    -   The **adapter** parameter can be used to specify the adapter.
-    -   The **port** parameter can be used to specify the local port to bind to.
+  - The *host* part specifies the remote host to contact.
+  - The *port* part specifies **both local and remote port**. Note that the local port is this way both listening port and outgoing port.
+  - The **adapter** parameter can be used to specify the adapter.
+  - The **port** parameter can be used to specify the local port to bind to.
 
-Some parameters handled for SRT medium are specific, all others are socket options. The following parameters are handled special way by *srt-live-transmit*:
+Some parameters handled for SRT medium are specific, all others are socket options. The following parameters are handled special way by `srt-live-transmit`:
 
 - **mode**: enforce caller, listener or rendezvous mode
 - **port**: enforce the **outgoing** port (the port number that will be set in the UDP packet as a source port when sent from this host). This can be used only in **caller mode**.
 - **blocking**: sets the `SRTO_RCVSYN` for input medium or `SRTO_SNDSYN` for output medium
 - **timeout**: sets `SRTO_RCVTIMEO` for input medium or `SRTO_SNDTIMEO` for output medium
 - **adapter**: sets the adapter for listening in *listener* or *rendezvous* mode
 
-All other parameters are SRT socket options. Here are some most characteristic options:
-
-- **latency**: Sets the maximum accepted transmission latency and should be >= 2.5 times the RTT (default: 120ms; when both parties set different values, the maximum of the two is used for both)
-- **passphrase**: Sets the password for the encrypted transmission.
-- **pbkeylen**:  Crypto key len in bytes {16,24,32} Default: 16 (128-bit)
-- **tlpktdrop**: Whether to drop packets that are not delivered on time. Default is on.
-- **conntimeo**: Connection timeout (in ms). Caller default: 3000, rendezvous (x 10)
-
-For the complete list of options, please refer to the SRT header file `srt.h` and search for `SRT_SOCKOPT` enum type. Please note that the set of available options may be version dependent. All options are available under the lowercase name of the option without the `SRTO_` prefix. For example, `SRTO_PASSPHRASE` can be set using
-a **passphrase** parameter. The mapping table `srt_options` can be found in `common/socketoptions.hpp` file.
+All other parameters are SRT socket options.
+
+| URI param   | SRT Option | Description | Possible values |
+| ----------- | ---------- | ----------- | --------------- |
```

Comment:
1. I think the column markers should be aligned to one size. Not every reader will be using markdown rendering.
2. The options should be sorted alphabetically.
3. The list of possible types should be predefined. Note that `bool` type allows for 3 specifications - `yes/no`, `on/off`, `true/false`, plus obviously `1/0` in every case when the option type is designated as BOOL, and this should be mentioned here. The "enumerable" values are ok as they are here, although no special markers are required. The requirements for positive-only values better be marked some universal way (note that, for example, both `fc` and `maxbw` require a positive integer, but you have mentioned this limitation only for the latter).
4. For cases when the type is unusual or some special values are involved, you could use annotations, expanded below the table.

Reply:
5. For some options at least you might specify defaults, especially this is important for boolean options. Optional default you could specify by adding `= value` expression in the type column.